Dockerfile

```diff
@@ -0,0 +1,15 @@
+
+# syntax=docker/dockerfile:1
+FROM python:3.8-slim-buster
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+COPY ./source .
+RUN python manage.py migrate
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
It's a mistake to run Django apps using internal web server, the right choise is to use uswgi or gunicorn so please use one from them instead